umm-spec-lib/src/cmr/umm_spec/migration/version/granule.clj

```diff
@@ -203,3 +211,20 @@
       (assoc :MetadataSpecification {:URL "https://cdn.earthdata.nasa.gov/umm/granule/v1.6.2"
                                      :Name "UMM-G"
                                      :Version "1.6.2"})))
+
+(defmethod interface/migrate-umm-version [:granule "1.6.3" "1.6.2"]
+  [context g & _]
+  (-> g
+      (specify-metadata "1.6.2")
+      (update :RelatedUrls
+              (fn [related-url]
```

Comment:
nit: move this into its own function and the argument should be named `related-urls`.